Makefile.am

```diff
@@ -214,7 +214,7 @@ install-data-local:: $(WEBPACK_DEBUG)
 	$(MKDIR_P) $(DESTDIR)$(debugdir)$(pkgdatadir)
 	tar -cf - $^ | tar -C $(DESTDIR)$(debugdir)$(pkgdatadir) --strip-components=1 -xvf -
 uninstall-local::
-	find $(DESTDIR)$(pkgdatadir) -ignore_readdir_race -type f -delete
+	find $(DESTDIR)$(pkgdatadir) -ignore_readdir_race -type f -exec rm -f {} \;
```

Comment:
Bah, so `-delete` does not respect `-ignore_readdir_race`? In my experiments for commit 5a398616b4fa that helped a lot. But yeah, rhetorical question, since it demonstrably doesn't.